db/re/item_randomopt_group.yml

```diff
@@ -12383,3 +12383,615 @@ Body:
             MinValue: 1
             MaxValue: 5
             Chance: 769
+  # Source: https://ro.gnjoy.com/news/probability/View.asp?category=6&seq=4136330
+  # Slot 2 should only have 33% chance to be applied
```

Comment:
Guess we dont have the 33% feature yet, but if you set MaxRandom to 2 and put the info of the second slot into the Random section it would at least not happen 100% of the time.

@Atemo, @secretdataz, @Everade what do you guys think? Any thoughts if and how we should adjust the database features/design to be able to support this?

Reply:
I suggest to define chance for **Slot** directly, and default its value to 10000 (100%)
So this example would result in:
```
    Slots:
      - Slot: 2
        Chance: 3300
        Options:
```

Reply:
yeah it could be like Everade said, or the chance could be defined in the laphine file eventually

Reply:
Requires #6864 to properly work.